spyder/plugins/editor/tests/test_plugin.py

```diff
@@ -63,36 +80,40 @@ def test_basic_initialization(setup_editor):
 
 
 @pytest.mark.parametrize(
-    'filenames, expected_filenames, splitsettings, expected_splitsettings', [
-        (['/file1.py', '/file2.py', '/file3.py', '/file4.py'],
-         ['/file1.py', '/file2.py', '/file3.py', '/file4.py'],
-         [(False, '/other_file.py', [2, 4, 8, 12])],
-         [(False, '/file1.py', [2, 4, 8, 12])]),
-        (['/file1.py', '/file2.py', '/file3.py', '/file4.py'],
-         ['/file2.py', '/file1.py', '/file3.py', '/file4.py'],
-         [(False, '/file2.py', [2, 4, 8, 12])],
-         [(False, '/file2.py', [4, 2, 8, 12])]),
-        (['/file1.py', '/file2.py', '/file3.py', '/file4.py'],
-         ['/file1.py', '/file2.py', '/file3.py', '/file4.py'],
-         None,
-         None, )
-    ])
-def test_reorder_filenames(setup_editor, expected_filenames, filenames,
```

Comment:
This test seems more comprehensive than the test you're replacing it with. Please explain why you're replacing it.

Reply:
I'm not actually replacing it. It is more like I'm removing `test_reorder_filenames` that was checking only the output of the `reorder_filenames` method and I'm creating a new test that is checking the results of `setup_open_files` instead. So in fact, I think the new test is more comprehensive because it actually checks the resulting order of the files in the `Editorstack` and also check that the right file is focused.

The only thing I do not cover compared to the old test, is when `splitsettings` is `None`, because unless I'm mistaken and I can be, this case is not supposed to happen at this point in the setup of Spyder, so there is no point in testing it.

I simplified the `@pytest.mark.parametrize` because some of the arguments were not needed and there was also duplication of information. Maybe that is why this test looks less comprehensive?

Reply:
Ok, thanks for the explanation.

Reply:
No problem. Thanks for the review, it is always appreciated.